probe/src/datagram_pb2.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this file the same as protocols/datagram_pb2.py? If so, can this just be a symlink? We should avoid committing duplicate code to the repo.

Reply:
imported from protocols folder now